R/slurm_commands.R

```diff
@@ -6,7 +6,7 @@
 #' @param user_name Username at the Nuvolos cluster, leave this empty from a Nuvolos application
 #' @param array If running an array job, a 2-length vector specifying starting and ending array indices
 #' @export
-sbatch <- function(script, n_cpus=4, queue="intq", use_mpi=FALSE, use_gsl = FALSE, user_name = NULL, array=NULL) {
+sbatch <- function(script, n_cpus=4, n_nodes=1, queue="intq", use_mpi=FALSE, use_gsl = FALSE, user_name = NULL, array=NULL) {
```

Comment:
Please put it as last parameter to avoid breaking if someone called things positionally so-far